Casks/a/aegisub.rb

```diff
@@ -1,14 +1,19 @@
 cask "aegisub" do
-  version "3.2.2"
-  sha256 "d71fa46f074a2d5a252f30779e0b8d913d5157258f5d9fc333411f8c9493f42b"
+  version "3.4.1"
+  sha256 "006f69f117552a071503f723ad9ac8685e8c051055dcb132e6409c9a2f4cde64"
 
-  url "https://github.com/Aegisub/Aegisub/releases/download/v#{version}/Aegisub-#{version}.dmg"
+  url "https://github.com/TypesettingTools/Aegisub/releases/download/v#{version}/Aegisub-#{version}.dmg"
   name "Aegisub"
   desc "Create and modify subtitles"
-  homepage "https://github.com/Aegisub/Aegisub"
+  homepage "https://github.com/TypesettingTools/Aegisub"
 
-  # https://github.com/Aegisub/Aegisub/issues/336
-  deprecate! date: "2024-09-06", because: :unmaintained
+  # The repo has a garbage "r6962" tag from 2012 that should be excluded
+  livecheck do
+    strategy :git do |tags|
+      # example: v3.4.1
+      tags.filter_map { |tag| tag[/^v?(\d+(?:\.\d+)*)$/i, 1] }
+    end
+  end
```

Comment:
The `url` is from a release artifact, so it is better to check latest release not tags.
```suggestion
  livecheck do
    url :url
    strategy :github_latest
  end
```

Reply:
Ah good nuance there, that makes sense. Thanks.